### DISCUSSION TITLE
Sailing Mode + auto discharge. yes/no

### DISCUSSION THREAD

**Discussion by FreemanGT** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1203] [Q&A]
Hey guys!

new pro user here

I'm a little confused about the settings, I have a feeling like auto discharge somehow conflicts with Sailing Mode, am I right?

is this a good setting for a brand-new MacBook Pro M3? 

thanks very much!!

![CleanShot 2024-04-03 at 18 23 14](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/165934120/cb173210-2f48-4217-b589-1ee77775fd17)
![CleanShot 2024-04-03 at 18 23 01](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/165934120/e17e8747-1dfc-46f0-a674-206a2bb3becb)

**Comment by MatthiasKerbl** [comment]
Hi @FreemanGT,

Automatic Discharge and Sailing Mode do not conflict with each other and can both be activated at the same time.

Best,
Matthias

**Reply by FreemanGT** [reply]
thanks very much!

are these settings good for my brand-new M3 MacBook pro in your opinion?

**Reply by MatthiasKerbl** [reply]
Yes, I have personally enabled both features for my MacBook and use case.

Best,
Matthias

**Comment by FreemanGT** [comment]
awesome thanks my friend

‫בתאריך יום ו׳, 5 באפר׳ 2024 ב-10:49 מאת ‪Matthias Kerbl‬‏ <‪
***@***.***‬‏>:‬
